src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -76,18 +80,39 @@ def trigger(*argv):
     if "config" not in args:
         msg = "no --config specified"
         raise KeyError(msg)
-
     task_name = args.get("task", "not specified")
+    job_id = args.get("job_id")
+    run_id = dbutils.widgets.get("run_id")
+    task_key = args.get("task_id")
     if task_name not in _TASKS:
         msg = f'task "{task_name}" not found. Valid tasks are: {", ".join(_TASKS.keys())}'
         raise KeyError(msg)
 
     current_task = _TASKS[task_name]
     print(current_task.doc)
 
+    config_path = Path(args["config"])
+    cfg = WorkspaceConfig.from_file(config_path)
     _install()
 
-    cfg = WorkspaceConfig.from_file(Path(args["config"]))
-    logging.getLogger("databricks").setLevel(cfg.log_level)
-
-    current_task.fn(cfg)
+    logger = logging.getLogger("databricks")
+    logger.setLevel(cfg.log_level)
```

Comment:
Can you configure it the way that log in the file always has the debug+ lines, even if standard output logger does have only INFO+?..

Reply:
Done